views/stylesheets/application.scss

```diff
@@ -2,12 +2,13 @@
 
 $black: black;
 $white: white;
-$primary: #4793e3;
+$primary: #4792e3;
```

Comment:
Why did this change?

Reply:
I just changed the background color a bit.

Reply:
I changed the background color slightly.